pkg/adt/interval_tree.go

```diff
@@ -425,14 +475,14 @@ func (ivt *IntervalTree) find(ivl Interval) (ret *intervalNode) {
 		ret = n
 		return false
 	}
-	ivt.root.visit(&ivl, f)
+	ivt.root.visit(&ivl, ivt.nilNode, f)
 	return ret
 }
 
 // Find gets the IntervalValue for the node matching the given interval
 func (ivt *IntervalTree) Find(ivl Interval) (ret *IntervalValue) {
 	n := ivt.find(ivl)
```

Comment:
If not found, does find() return nil or nilNode?

Reply:
find() function return is nilNode, it is private function, I think its ok.

please help me check the code